README.md

```diff
@@ -1,2 +1,10 @@
 # ffsl_manager
-Node.js for the FFSL -- the world's greatest fantasy football league.
+Backend for the FFSL -- the world's greatest fantasy football league.
+
+## Database
+`cd database`
```

Comment:
I know this is just a readme, but does this actually do anything or is it just a list of instructions?

Reply:
It's just a list of instructions there is no automation but I could throw them into a bash file if you wanted but I'm assuming you're on windows.